test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -299,7 +305,52 @@ public void PlattCalibratorOnnxConversionTest()
             Done();
         }
 
-        class PlattModelInput
+        [Fact]
+        public void FixedPlattCalibratorOnnxConversionTest()
+        {
+            var (mlContext, dataView, estimators, initialPipeline) = GetEstimatorsForOnnxConversionTests();
+            foreach (var estimator in estimators)
+            {
+                // Utilize FixedPlattCalibrator by defining slope and offset
+                var pipeline = initialPipeline.Append(estimator).Append(mlContext.BinaryClassification.Calibrators.Platt(slope: -1f, offset: -0.05f));
+                var onnxFileName = $"{estimator}-WithFixedPlattCalibrator.onnx";
+
+                TestPipeline(pipeline, dataView, onnxFileName, new ColumnComparison[] { new ColumnComparison("Score", 3), new ColumnComparison("PredictedLabel"), new ColumnComparison("Probability", 3) });
+            }
+            Done();
+        }
+
+        [Fact]
+        [Trait("Category", "SkipInCI")]
+        public void NaiveCalibratorOnnxConversionTest()
+        {
+            var (mlContext, dataView, estimators, initialPipeline) = GetEstimatorsForOnnxConversionTests();
+            foreach (var estimator in estimators)
+            {
+                var pipeline = initialPipeline.Append(estimator).Append(mlContext.BinaryClassification.Calibrators.Naive());
+                var onnxFileName = $"{estimator}-WithNaiveCalibrator.onnx";
+
+                TestPipeline(pipeline, dataView, onnxFileName, new ColumnComparison[] { new ColumnComparison("Score", 3), new ColumnComparison("PredictedLabel"), new ColumnComparison("Probability", 3) });
+            }
+            Done();
+        }
+
+        [Fact]
+        [Trait("Category", "SkipInCI")]
+        public void IsotonicCalibratorOnnxConversionTest()
+        {
+            var (mlContext, dataView, estimators, initialPipeline) = GetEstimatorsForOnnxConversionTests();
+            foreach (var estimator in estimators)
+            {
+                var pipeline = initialPipeline.Append(estimator).Append(mlContext.BinaryClassification.Calibrators.Isotonic());
+                var onnxFileName = $"{estimator}-WithIsotonicCalibrator.onnx";
+
+                TestPipeline(pipeline, dataView, onnxFileName, new ColumnComparison[] { new ColumnComparison("Score", 3), new ColumnComparison("PredictedLabel"), new ColumnComparison("Probability", 3) });
+            }
+            Done();
+        }
+
+        class ModelInput
         {
```

Comment:
Do we need both ModelInput and PlattModelInput2? Is it possible to also merge PlattGetData and PlattGetData2?

Reply:
`PlattModelInput2` and `PlattGetData2()` are used to test ONNX export of `PlattCalibrator` with non-standard model inputs such as score column being named "ScoreX" instead of the default name "Score". I will rename `PlattModelInput2' to add non-standard model input ONNX export testing for the other calibrators as well.